assets/dev/scss/frontend/general/_button.scss

```diff
@@ -83,9 +92,27 @@
 
 	span {
 		text-decoration: inherit; //fix for conflict with text-decoration & inline-block
+		align-self: center;
+	}
+
+	&:hover {
+
+		.elementor-button-icon {
+
+			i {
+				color: var(--button-icon-color-hover);
+				transition: all 0.3s;
```

Comment:
I assume this styling should be set by the control? And not hardcoded like this?

Reply:
@Mchris-hub I will need feedback.